### ISSUE TITLE
SPIKE: Consider accessibility of popovers

### ISSUE THREAD

**Issue by justinlittman** [LD4P/sinopia_editor#3243]
See discussion in #3219

For this ticket, determine whether current implementation of popovers is accessible, including links within a popover. If current implementation is not accessible, propose how to remediate.

**Comment by peetucket** [comment]
Started looking at other potential libraries that could be used for popovers, starting with this one, which says it creates accessible popovers out of the box:

https://www.npmjs.com/package/@accessible/popover

It does introduce additional dependencies however.  Styling would need some work, but that seems manageable.  Current issue is that I cannot get it to render HTML within the popover it creates (which is required if a user has a link inside the remark).

**Comment by justinlittman** [comment]
Based on reviewing https://getbootstrap.com/docs/5.1/components/popovers/#usage, having considered other options, and lacking clarity on whether there is, in fact, an accessibility problem, I'm going to close this ticket.

**Comment by jcoyne** [comment]
@justinlittman there is in fact an accessibility problem and I can demonstrate it to you. Just give me 5 minutes after standup.

**Comment by justinlittman** [comment]
Much appreciated.
